bgs/blobs.go

```diff
@@ -0,0 +1,29 @@
+package bgs
```

Comment:
Should you pull this out into a top-level `blobstore` package? Seems like it could be reused by the PDS.

Reply:
Yeah, that makes sense.